specs/regression.txt

```diff
@@ -708,3 +708,25 @@ ISSUE 399
 the size of <code>usize</code> and have the same alignment.</p>
 </blockquote>
 ````````````````````````````````
+
+ISSUE 413
+
+```````````````````````````````` example
+Lorem ipsum.[^a]
+
+An unordered list before the footnotes:
+* Ipsum
+* Lorem
+
+[^a]: Cool.
+.
+<p>Lorem ipsum.<sup class="footnote-reference"><a href="#a">1</a></sup></p>
+<p>An unordered list before the footnotes:</p>
+<ul>
+    <li>Ipsum</li>
+    <li>Lorem</li>
+</ul>
+<div class="footnote-definition" id="a"><sup class="footnote-definition-label">1</sup>
+    <p>Cool.</p>
+</div>
+````````````````````````````````
```

Comment:
When we have examples/tests specifically related to extra features like footnotes, is there a way we can mark them all so they can be reviewed more easily if the footnote spec is updated in the future?

Reply:
There currently is not, but that would be super useful. It should be easy enough to put these in separate files.